examples/research_presentation/sota_review.py

```
@@ -0,0 +1,69 @@
+    Args:
+        topic_or_contents: The topic (or detailed contents) for the presentation.
+
+    Returns:
+        The file path to the PowerPoint slide deck.
+    """
+    try:
+        from slidedeckai.core import SlideDeckAI
+    except ImportError as e:
+        raise ImportError(
+            'Please install the `slidedeckai` package to use this tool: pip install slidedeckai'
+        ) from e
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)